### ISSUE TITLE
Mistakes in you PINN.ipynb example.

### ISSUE THREAD

**Issue by ghost** [MKyasAI/ML_Training#1]
In your PINN example you suggest, that the `physics_loss` does not really help you neural network. 
This is actually wrong, there is just an error in the implementation. 

In your implementation you allways consider the first entries of size batch size. 
```
X_batch = X_velocity[:tf.shape(y_true)[0]]  # Adjust to the current batch size
```

You can somehow circumvent this problem by setting the `batch_size` to the `num_samples` and additionally set `shuffle=False`
```
history = pinn_model.fit(X_velocity, y_range_noisy, epochs=400, batch_size=200, shuffle=False, verbose=0, callbacks=[LossMonitor()])
```

With this setting you will see, the NN trains as good (maybe even slightly better) with the `physics_loss` only, compared to the `data_loss` only. 
This makes absolutely sense, since both are identical assuming no noise in the data.

**Comment by MKyasAI** [comment]
Thanks for opening the issue! I've updated the file with the workaround fix. The physics term works as expected now.
